src/protocols/bdx/TransferFacilitator.h

```diff
@@ -133,18 +111,12 @@ class Initiator : public TransferFacilitator
 {
 public:
     /**
-     * Initialize the TransferSession state machine to prepare a transfer request message (does not send the message) and start the
-     * poll timer.
+     * Initialize the TransferSession state machine to prepare a transfer request message and sends the outgoing message
```

Comment:
"prepare" but "sends"?  Hard to make sense of this sentence.

Reply:
same as above. not required

Reply:
done